smoke-tests/src/test/java/com/splunk/opentelemetry/AppServerTest.java

```diff
@@ -27,11 +25,7 @@
 import com.splunk.opentelemetry.helper.TestImage;
 import io.opentelemetry.proto.trace.v1.Span;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Did you accidentally change it to use wildcard imports?

Reply:
Hmm, it must have happened when I reset IntelliJ settings around the time groovy code formatting in the upstream repo was changed. Thanks for catching this!